### ISSUE TITLE
Support for 1.0

### ISSUE THREAD

**Issue by Roger-luo** [memoiry/lightML.jl#8]
Is this project abandoned? Any plans for 1.0?

**Comment by jlapeyre** [comment]
See [my fork](https://github.com/jlapeyre/LightML.jl) of this repository. In this fork, a large part of the package is compatible with Julia 1.0.

I have not issued PRs for any of these changes, but can do so if desired.

**Comment by jlapeyre** [comment]
On my fork, all of the examples run on v1.0. The repo was not originally designed to be a package, so the code is not organized like a package. 

I changed this a bit. But, it is just started.

1. Move data from ./src to ./resources/data
2. Rewrite some examples (two or three) as scripts that `use` the package and put them in ./examples.
3. Move or copy some code from examples to ./test.

This is all done in many commits, so they may be added selectively.

**Comment by memoiry** [comment]
@jlapeyre  Great job, thanks for you contribution!  
1. I haven't going around Julia Lang for a while so it may take me some time to pick up the new feature. Is there any plan that you may sent a pull request related to this?

2. Yes, The repository is not originally designed as a package, it's simply a collection of function and class for learning purpose, so the code structure is a little messy

Thanks!

**Comment by jlapeyre** [comment]
Yes. I'd be happy to make a pull request. I made some structural changes. You might not be interested in all of them..... Maybe I can start with just the bare minimum to get most things working with Julia 1.0
